external/avro/src/main/scala/org/apache/spark/sql/avro/AvroOptions.scala

```diff
@@ -40,6 +40,14 @@ class AvroOptions(
    */
   val schema: Option[String] = parameters.get("avroSchema")
 
+  /**
+   * Optional Avro schema (in JSON format) that was used to serialize the data.
+   * This should be set if the schema provided for deserialization is compatible
+   * with - but not the same as - the one used to originally convert the data to Avro.
+   * See https://github.com/apache/spark/pull/24405 for more details.
```

Comment:
I think note with jira number SPARK-27506, instead of pull request link, is more preferred and usual.